sandboxed-fs.js

```diff
@@ -4,11 +4,109 @@ const sandboxedFs = {};
 module.exports = sandboxedFs;
 
 const fs = require('fs');
+const pathModule = require('path');
+
+const errorMessage = 'path must be a string';
```

Comment:
It looks like they use lowercase first letter with `TypeError` but go for uppercase with `Error`.